install/functions.php

```diff
@@ -2427,6 +2409,8 @@ function update420_430()
     mysqli_query($_database, "INSERT INTO `" . PREFIX . "modrewrite` (`regex`, `link`, `fields`, `replace_regex`, `replace_result`, `rebuild_regex`, `rebuild_result`) VALUES('whoisonline.html#was','index.php?site=whoisonline#was','a:0:{}','index\\\\.php\\\\?site=whoisonline#was','whoisonline.html#was','whoisonline\\\\.html#was','index.php?site=whoisonline#was')");
     mysqli_query($_database, "INSERT INTO `" . PREFIX . "modrewrite` (`regex`, `link`, `fields`, `replace_regex`, `replace_result`, `rebuild_regex`, `rebuild_result`) VALUES('whoisonline/{sort}/{type}.html','index.php?site=whoisonline&sort={sort}&type={type}','a:2:{s:4:\"sort\";s:6:\"string\";s:4:\"type\";s:6:\"string\";}','index\\\\.php\\\\?site=whoisonline[&|&amp;]*sort=(\\\\w*?)[&|&amp;]*type=(\\\\w*?)','whoisonline/$3/$4.html','whoisonline\\\\/(\\\\w*?)\\\\/(\\\\w*?)\\\\.html','index.php?site=whoisonline&sort=$1&type=$2')");
 
+    mysqli_query($_database, "DROP TABLE `" . PREFIX . "settings`");
+
     updateMySQLConfig();
 }
 
```

Comment:
Das ganze sollte auch eigentlich so lauten:
mysqli_query($_database, "DROP TABLE `" . PREFIX . "styles`");

Allerdings sollte man vor dem löschen  vielleicht den Titel noch auslesen und in die title spalte der settings tabelle einfügen. Oder kann man per SQL auch irgend wie Tabellen Spalten in andere Tabellen verschieben. Ich glaube aber nicht.